### DISCUSSION TITLE
Feature request: add categories to GraphQL API in Icon section

### DISCUSSION THREAD

**Discussion by cwiesenekker** [FortAwesome/Font-Awesome#20898] [Feature Requests]
### Is your feature request related to a problem?

We want to build an icon picker which you can navigate through by categories. 

### Feature description

The problem is that we want to use a picker we used. We created on the graphql a script that imports all the icons with their categories. We used api-canary.fontawesome.com for that (you guys used that on your website too). But in the production api its missing that categories feature. Its very usefull to create and categorize icons on your own. 

So in the end i am able to do:
`query GetMeCategorizedIcons {
  release(version: "6.3.0") {
    icons {
      id
      label
      shim {
        id
        name
        prefix
      }
      categories
      styles
    }
  }
}
`
Can you please integrate that again :)

### Alternatives

We used to scrape your website to create a list in our own product... Which was kind of, lets say, optimisch to get it stable :D We dont want to use/do that anymore, but the API is missing some features for now.

### Feature request checklist

- [X] The title starts with "Feature request: " and is followed by the requested feature description
- [X] This is a single feature
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by ludoviclenoir** [comment]
I approve ! Please need this :)

**Comment by emilie-signelazer** [comment]
Would be so usefull! Absolutely needed!!!

**Comment by martinchloe** [comment]
I need this for a wysiwyg project too 🙏

**Comment by robmadole** [comment]
Thanks, all. We'll consider this!

**Comment by sebasounds** [comment]
+1

**Comment by cwiesenekkerzig** [comment]
> Thanks, all. We'll consider this!

It's actually been build already right :D maby release it again as some kind of labs feature 😇? Everything is better than what we do now
